###############################################################################
## Configuration file
## Drone name: simu
## Date: 08 January 2020
## Tuning:
## -fsf: yes
## -se : no
###############################################################################
ctrl_freq: 100

# fsf_x_P: 0.88
# fsf_x_S: 1.3
# fsf_y_P: 0.88
# fsf_y_S: 1.3
# #fsf_z_P: 0.90
# #fsf_z_S: 1.34
# fsf_z_P: 0.94
# fsf_z_S: 3.3

fsf_x_P: 1.98
fsf_x_S: 2.44
fsf_x_A: 1.0
fsf_y_P: 1.98
fsf_y_S: 2.44
fsf_y_A: 1.0
fsf_z_P: 1.98
fsf_z_S: 2.44
fsf_z_A: 1.0

use_StatesObserver: true

# se_x_Lpos: 0.75
# se_y_Lpos: 0.75
# se_z_Lpos: 0.75
# se_x_Lspeed: 2.4
# se_y_Lspeed: 2.4
# se_z_Lspeed: 2.4
# se_x_Lacc: 0.92
# se_y_Lacc: 0.92
# se_z_Lacc: 0.92
# se_x_Lunc: 3.0
# se_y_Lunc: 3.0
# se_z_Lunc: 3.0

se_x_Lpos: 0.67
se_y_Lpos: 0.67
se_z_Lpos: 0.67
se_x_Lspeed: 3.4
se_y_Lspeed: 3.4
se_z_Lspeed: 3.4
se_x_Lunc: 6.0
se_y_Lunc: 6.0
se_z_Lunc: 6.0

#se_x_Lpos: 1.03
#se_y_Lpos: 1.03
#se_z_Lpos: 1.03
#se_x_Lspeed: 10.7
#se_y_Lspeed: 10.7
#se_z_Lspeed: 10.7
#se_x_Lunc: 9.62
#se_y_Lunc: 9.62
#se_z_Lunc: 9.62

se_x_Filter: 0.9
se_y_Filter: 0.9
se_z_Filter: 0.1

mass: 1.5
hoverCompensation: 0.45
maxAngle: 45.0
maxVerticalAcceleration: 4.0
compensateYaw: true
Ky: 0.5
